sale_margin_percentage/i18n/es_CR.po

```diff
@@ -0,0 +1,107 @@
+# Translation of Odoo Server.
```

Comment:
`es.po`, please.

Country-specific languages should only be included when their terms differ from the base ones. In addition, you deleted `es.po`